lib/routes/account.js

```diff
@@ -946,10 +1013,33 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+
+          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
+
+          if (doSigninConfirmation) {
+            return db.sessionTokenWithVerificationStatus(sessionToken.id)
+              .then(function (token) {
+
+                var sessionVerified = token.tokenVerified
```

Comment:
```
{
 verified: "email verified value",
 sessionVerified: "session verified value"
}
```

Unfortunately, that won't work. All existing clients look only at the "verified" field to decide whether to show a "go verify your email" screen, which we want to happen if either the email or session need to be verified. The browser also looks at that field to determine whether it needs to display the "Go verify your email" button in about:preferences#sync and the warning sign in the hamburger menu.